packages/plugin-ext/src/common/connection.ts

```diff
@@ -52,23 +55,28 @@ export class PluginConnection implements Connection {
 }
 
 /**
- * [IWebSocket](#IWebSocket) implementation over RPC.
+ * Wrapper around a [PluginConnection](#PluginConnection) to match the [Channel](#Channel) interface.
  */
-export class PluginWebSocketChannel implements IWebSocket {
-    constructor(protected readonly connection: PluginConnection) { }
+export class PluginChannel implements Channel<string> {
```

Comment:
We're wrapping a string channel around a json rcp connection which wraps a string channel. That does not really make sense, IMO. I got rid of that all in https://github.com/eclipse-theia/theia/pull/10333. So I wouldn't sweat the small stuff here.

Reply:
I did what I must to get it to compile as before, I'd be more than happy to merging #10333 after this!